### ISSUE TITLE
Windows compatibility issue

### ISSUE THREAD

**Issue by mischakolbe** [portato-io/portatoapp#123]
Are these fancier start/build/test scripts in the package.json required?
```
...
  "scripts": {
    "start": "REACT_APP_COMMIT_HASH=$(git rev-parse HEAD) react-scripts start",
    "build": "REACT_APP_COMMIT_HASH=$(git rev-parse HEAD) react-scripts build",
    "test": "REACT_APP_COMMIT_HASH=$(git rev-parse HEAD) react-scripts test",
    ...
```

I got errors on Windows (terminal, bash & powershell) and a hacky equivalent probably doesn't work on Mac...
```
...
  "scripts": {
    "start": "git rev-parse HEAD >sha.txt && set /p REACT_APP_COMMIT_HASH= < sha.txt && del sha.txt && react-scripts start",
    "build": "git rev-parse HEAD >sha.txt && set /p REACT_APP_COMMIT_HASH= < sha.txt && del sha.txt && react-scripts build",
    "test": "git rev-parse HEAD >sha.txt && set /p REACT_APP_COMMIT_HASH= < sha.txt && del sha.txt && react-scripts test",
    ...
```

I don't think the commit hash is very necessary and we could just switch to this?
```
...
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    ...
```

**Comment by mAkeddar** [comment]
Yep the original file for scripts is this one 
"scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
 }
 
 The app launch doesn't depend on the hash, right @hugo2410 ?

**Comment by hugo2410** [comment]
@mAkeddar @mischakolbe having the git hash before building allows us to have for every single build. So, we are sure that even if we get the versioning wrong (as it is done manually) we are able to trace back to the specific version of the code. It would allow us to debug easier. I can try and investigate how we need to adapt the path to make it work cross-platform

**Comment by mischakolbe** [comment]
Fair point, Hugo. Good to know! If it gets embedded into the build to make
it reproducible it's another story. At first glance it looked like it was
just for a printout when running npm start.
